### PR TITLE
tracing: Simplified creation of spans.

### PR DESCRIPTION
**Pull request by bwplotka**
cc @s-urbaniak (: 

Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: pkg/store/bucket.go [hunk 1]
```diff
@@ -962,16 +958,18 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_Serie
 			}
 
 			if err := srv.Send(storepb.NewSeriesResponse(&series)); err != nil {
-				return status.Error(codes.Unknown, errors.Wrap(err, "send series response").Error())
+				err = status.Error(codes.Unknown, errors.Wrap(err, "send series response").Error())
```
Comment (s-urbaniak) [inline pkg/store/bucket.go]: i believe this still needs a `return`, no?

Comment (s-urbaniak) [reply pkg/store/bucket.go]: scratch that, you need a capturing `var err error` before the `if` block :)

File: pkg/store/bucket.go [hunk 2]
```diff
@@ -962,16 +958,18 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_Serie
 			}
 
 			if err := srv.Send(storepb.NewSeriesResponse(&series)); err != nil {
```
Comment (s-urbaniak) [inline pkg/store/bucket.go]: sorry, i was blind, i guess this just needs `if err = srv.Send...`

Comment (s-urbaniak) [reply pkg/store/bucket.go]: otherwise the shadowed `err` will be an ineffectual assignment as CI complained :-)

**Comment by s-urbaniak** [comment]
We just need to fix two shadowed `err`s, else lgtm! :tada:

File: pkg/store/bucket.go (hunk 2 above)
Comment (bwplotka) [reply pkg/store/bucket.go]: Wow! Thanks `go vet` :+1:

**Pull request merged**